### DISCUSSION TITLE
調用expand_translator.lua，出現執行”Memory“ call 的錯誤

### DISCUSSION THREAD

**Discussion by JoKlever** [rime/home#1348] [Q&A]
試著調用 **expand_translator.lua**，出現錯誤："E0725 18:49:31.638553 38108 lua_gears.cc:44] Lua error:(2): ...s\General\AppData\Roaming\Rime\lua\expand_translator.lua:39: attempt to call a nil value (global '**Memory**')"。應該是在local function init(env)裡，要執行的“Memory" function 不存在。請問各位高手是否有用過這script，請求指點迷津。

Thanks in advance!

**Comment by MokOopsing** [comment]
librime-lua版本不够高吗

**Reply by JoKlever** [reply]
我是用 lua 5.4 (x64 on Window 10)。 才開始試Rime和這語言，還很生疏。看起來“Memory”不像是lua內建模組或關鍵字，可能需要從別處導入（via “require” maybe）。其它範例也都沒用到“Memory"這方法/函數。

感謝回复！

**Reply by MokOopsing** [reply]
我说的是librime-lua插件的版本，不是你电脑上装的lua版本
